cognite/client/_api_client.py

```diff
@@ -65,7 +64,7 @@
 
 class APIClient:
     _RESOURCE_PATH: str
-    _RETRYABLE_POST_ENDPOINT_REGEX_PATTERNS: ClassVar[set[str]] = {
+    _RETRYABLE_POST_ENDPOINT_REGEX_PATTERNS: frozenset[str] = frozenset(
```

Comment:
Should still be annotated as a ClassVar.
```suggestion
_RETRYABLE_POST_ENDPOINT_REGEX_PATTERNS: ClassVar[frozenset[str]] = frozenset(
```